### PR TITLE
bugfix: update old links

### PR DESCRIPTION
**Pull request by endersonmenezes**
Co-authored-by: Guilherme Siquinelli guiseek@gmail.com

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -193,9 +193,9 @@ <h2 id="liaisons">
     <ul>
       <li>
         <a href=
-        "https://github.com/whatwg/sg/blob/master/Workstreams.md#html">HTML</a>
+        "https://whatwg.org/workstreams#html">HTML</a>
```
Comment (yoavweiss) [inline charter.html:196]: The new workstreams page doesn't include IDs for the different streams, so these anchors don't resolve.

Comment (endersonmenezes) [reply charter.html:196]: We fixed this in: https://github.com/whatwg/whatwg.org/commit/839ca039f8e5b8c7ab9c2e04dafb18282f45b4d6

I'll check what happened on a new build that removed it. Please keep this PR open.

@yoavweiss

Comment (endersonmenezes) [reply charter.html:196]: image

It's there buddy!

Comment (yoavweiss) [reply charter.html:196]: Awesome, thanks!! :)

**Review by yoavweiss — APPROVED** [review]
LGTM

**Comment by yoavweiss** [comment]
Thanks for fixing those links!! :)

**Pull request merged**